tests/terraform/parser/resources/parser_scenarios/tobool_function/expected.json

```diff
@@ -6,7 +6,7 @@
         "STRING_T": [true],
         "RAW_F": [false],
         "STRING_F": [false],
-        "JUNK": ["${tobool(\"invalid\")}"]
+        "JUNK": ["\"invalid\""]
```

Comment:
This is certainly a valid option, but I don’t know if it’s the best thing to convert to. In terraform, this would generate an error. My approach has generally otherwise been to avoid changing anything is a value couldn’t be resolved. So, personally, I have a slight preference to leave the statement unresolved.